packages/office-ui-fabric-react/src/components/Shimmer/ShimmerGap/ShimmerGap.styles.ts

```diff
@@ -0,0 +1,39 @@
+import { IShimmerGapStyleProps, IShimmerGapStyles } from './ShimmerGap.types';
+import { IStyleSet, getGlobalClassNames, HighContrastSelector } from '../../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-ShimmerGap-root'
+};
+
+export function getStyles(props: IShimmerGapStyleProps): IShimmerGapStyles {
+  const { height, widthInPercentage, widthInPixel, borderStyle, theme } = props;
+
+  const { palette } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const styles: IStyleSet = !!borderStyle ? borderStyle : {};
+  const ACTUAL_WIDTH = widthInPercentage ? widthInPercentage + '%' : widthInPixel ? widthInPixel + 'px' : '10px';
+
+  return {
+    root: [
+      classNames.root,
+      styles,
+      {
+        backgroundColor: palette.white,
+        width: ACTUAL_WIDTH,
+        minWidth: widthInPixel ? ACTUAL_WIDTH : 'auto', // Fix for IE11 flex items
```

Comment:
Would be be better to do this using inline styling?

Reply:
Fixed!